packages/cf-component-page/src/PageHeader.js

```diff
@@ -6,7 +6,7 @@ class PageHeader extends React.Component {
       <header className="cf-page__header">
         <h1 className="cf-page__title">{this.props.title}</h1>
         {this.props.subtitle &&
-          <p className="cf-page__subtitle">{this.props.subtitle}</p>}
+          <h2 className="cf-page__subtitle">{this.props.subtitle}</h2>}
```

Comment:
I think we should avoid it whenever possible.  However we have instances where our components depend on each other.  `cf-component-card` has a dependency on `cf-component link` for example.

If we expect this `<h2>` to look and function the same as `cf-component-heading` we should introduce that dependency here since not doing so makes this use a customization that will limit its ability to get future updates we make to that component.  This has technically already happened since `cf-component-heading` has been converted to Fela but this component is still using hard coded CSS classes.